### PR TITLE
Small fixes to website and adding contributing info

### PR DESCRIPTION
**Pull request by rahulkjoshi**
- Added section to contributing doc explaining requirements to locally test docs.
- Fixed broken links
- Added diagram to API explanation
- Cross-linked example API objects to user stories

### CODE REVIEW

File: site-src/api-overview.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ Currently, there are two main objects in the Network Policy API resource model:
 
 - **BaselineAdminNetworkPolicy (BANP)**
 
+![Alt text](./images/ANP-api-model.png?raw=true "Admin Network Policy API Model")
```
Comment (astoycos) [inline site-src/api-overview.md:36]: Can you ad a note here just something along the lines of 

"The below diagram demonstrates how these new API objects interact with each-other and existing NetworkPolicy Objects."

File: site-src/user-story-examples/user-story-3.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+metadata:
+  name: pub-svc-delegate-example
+spec:
+  priority: 20
+  subject:
+    namespaces: {}
+  egress:
+  - action: Pass
+    to:
+    - namespaces:
```
Comment (astoycos) [inline site-src/user-story-examples/user-story-3.yaml:12]: @tssurya Discovered this was wrong :) This should be something like

File: site-src/user-story-examples/user-story-3.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        namespaceSelector:
+          matchLabels:
+            kubernetes.io/metadata.name: bar-ns-1
+      pods:
+        podSelector:
+          matchLabels:
+            app: svc-pub
```
Comment (astoycos) [inline site-src/user-story-examples/user-story-3.yaml:19]:
```suggestion
    - pods:
        namespaceSelector:
          matchLabels:
            kubernetes.io/metadata.name: bar-ns-1
        podSelector:
          matchLabels:
            app: svc-pub
```

File: site-src/user-story-examples/user-story-3.yaml (hunk 2 above)
Comment (astoycos) [reply site-src/user-story-examples/user-story-3.yaml:12]: Are there any other issues you found @tssurya?

**Comment by astoycos** [comment]
Thanks for doing this @rahulkjoshi Just a few questions!

File: site-src/api-overview.md (hunk 1 above)
Comment (rahulkjoshi) [reply site-src/api-overview.md:39]: done!

**Comment by rahulkjoshi** [comment]
Thanks for the poke, I lost some momentum here 🙃 

PTAL, I've rebased and addressed the comments.

File: site-src/user-story-examples/user-story-3.yaml (hunk 3 above)
Comment (astoycos) [reply site-src/user-story-examples/user-story-3.yaml]: Thanks

**Comment by astoycos** [comment]
Thanks @rahulkjoshi 

/lgtm
/approve

**Pull request merged**